application/forms/Config/BackendForm.php

```diff
@@ -45,6 +45,14 @@ public function createElements(array $formData)
                     'label'             => $this->translate('Graphite Web password'),
                     'description'       => $this->translate('The above user\'s password')
                 ]
+            ],
+            [
+                'checkbox',
+                'graphite_insecure',
+                [
+                    'label'         => $this->translate('Connect insecurely'),
+                    'description'   => $this->translate('Check this not to verify the remote\'s TLS certificate')
```

Comment:
I think this should read "Check this to not..."

Reply:
As discussed w/ @gunnarbeutner: You're right.